### ISSUE TITLE
Ratings continual loading - Uncaught type error

### ISSUE THREAD

**Issue by Supergran** [XoopsModules25x/publisher#47]
Trying to get to grips with the Publisher module for content.  The ratings stars show up and highlight on hover but clicking on them results in a continuous loading which never resolves.  No php errors showing but the console shows an uncaught type error: Cannot set property 'innerHTML' of null
relates to the rating.js line 92
        var viewer = document.getElementById(div2show);
        viewer.innerHTML = text;
Presumably the scriptt can't find the property?
JAn

**Comment by mambax7** [comment]
What PHP and XOOPS version, which Publisher version? I've uploaded yesterday some updates, so please try the latest version. 
I was trying on my PC and everything was working fine here...

**Comment by Supergran** [comment]
Last Release 2.5.8 Michael. I'll try it again on the latest files and get back to you. Php 7.06. And bootstrap.

**Comment by Supergran** [comment]
Works fine on the latest files. :)
 The shareaholic script was also not functioning and that now works also.  The first load with bootstrap theme looks much better.  It's impressive to see all of this coming together.  Thank-you. Excellent job.

**Comment by mambax7** [comment]
In order to see the extra tabs for the article, you have to set the permissions for the Webmaster for all categories.

**Comment by Supergran** [comment]
Hi and thank you for such a quick response.  I set it  for all permissions for webmasters and still had the problem.  Just checked and they were all ticked....added them all again and now they're there!  Maybe a cache issue although I did try clearing  xoops and browser cache and have all modules to no cache.  Anyway...I'm now up and running with Publisher and the tutorial is a great example for me to use with the  pedigree module.  Thank you very much Michael.  I am really starting to realise just how much is possible using an xoops set up although I also think the more I learn the more I realise I have yet to learn!
